parsers/parser_utils.py

```diff
@@ -16,6 +16,7 @@ def standard_entry_to_dict(article, source, tz, strip_description=False):
     else:
         article_dict['abstract'] = article['description']
     od = collections.OrderedDict(sorted(article_dict.items()))
+    # The bug of the key??
```

Comment:
I think we should fix the bug in the feature extractors and not here. 
I have already fixed it in my branch